### PR TITLE
[FEATURE]JS scripts

### PR DESCRIPTION
**Pull request by ayush-raj13**
Dummy data for given no. of rows in all models can be generated using applicant.bat and recruiter.bat

### CODE REVIEW

File: javascripts/applicantDummyData.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
+/* eslint-disable */
```
Comment (kole-swapnil) [inline javascripts/applicantDummyData.js:1]: remove

File: javascripts/applicantDummyData.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    4,
+  );
+  const user = {
+    username,
+    email,
+    password: 'password',
+  };
+  const res = await axios.post('http://localhost:5000/api/v1/user/register', user, {
+    jar: cookieJar, // tough.CookieJar or boolean
+    withCredentials: true, // If true, send cookie stored in jar
```
Comment (kole-swapnil) [inline javascripts/applicantDummyData.js:30]: no proper indentation

File: javascripts/applicantDummyData.js [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    "Indian Institute of Science Education and Research, Kolkata",
+    "Indian Institute of Science Education and Research, Mohali",
+    "Indian Institute of Science Education and Research, Bhopal",
+    "Indian Institute of Science Education and Research, Tirupati",
+    "Indian Institute of Science Education and Research, Thiruvananthapuram",
+    "Indian Institute of Technology Mandi Kamand",
+    "Indian Institute of Technology Bhilai",
+    "Indian Institute of Technology Jammu"
+  ];
+
```
Comment (kole-swapnil) [inline javascripts/applicantDummyData.js:133]: make new file javascripts/constants.js
put all these arrays in there.
=> export const collegeList

File: javascripts/recruiterDummyData.js [hunk 4]
```diff
@@ -0,0 +1,263 @@
+/* eslint-disable */
+
```
Comment (kole-swapnil) [inline javascripts/recruiterDummyData.js:2]: remove

File: javascripts/recruiterDummyData.js [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+    "Hi, I'm Jason. With a focus on finding the right fit for both the candidate and the company, let's work together to achieve your career goals.",
+    "Hello, my name is Lauren. I specialize in placing candidates in the marketing and advertising industry. Let's connect and discuss your next move.",
+    "Hey, I'm Matthew. As a recruiter, I believe in being a partner in your career journey. Let's work together to find your dream job.",
+    "Hi, I'm Kelly. With a background in human resources, I have the expertise to help you succeed in your job search. Let's get started!",
+    "Hi, I'm Daniel. As a recruiter, I strive to provide a positive and supportive experience for all candidates. Let's connect and discuss your career aspirations.",
+    "Hello, I'm Megan. With a focus on building strong relationships, let me assist you in finding your next career move.",
+    "Hey there, I'm Brandon. I have experience placing candidates in a variety of industries, and I'm here to help you navigate the job market. Let's chat!"
+  ];
+  const ats_list = [
```
Comment (kole-swapnil) [inline javascripts/recruiterDummyData.js:82]: move to constants

**Review by kole-swapnil — CHANGES_REQUESTED** [review]

File: javascripts/applicantDummyData.js (hunk 1 above)
Comment (ayush-raj13) [reply javascripts/applicantDummyData.js]: done

File: javascripts/applicantDummyData.js (hunk 2 above)
Comment (ayush-raj13) [reply javascripts/applicantDummyData.js:29]: formatted through eslint

File: javascripts/applicantDummyData.js (hunk 3 above)
Comment (ayush-raj13) [reply javascripts/applicantDummyData.js]: done

File: javascripts/recruiterDummyData.js (hunk 4 above)
Comment (ayush-raj13) [reply javascripts/recruiterDummyData.js]: done

File: javascripts/recruiterDummyData.js (hunk 5 above)
Comment (ayush-raj13) [reply javascripts/recruiterDummyData.js]: done

File: javascripts/applicantDummyData.js [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+    email,
+    password: 'password',
+  };
+  const res = await axios.post('http://localhost:5000/api/v1/user/register', user, {
+    jar: cookieJar, // tough.CookieJar or boolean
+    withCredentials: true, // If true, send cookie stored in jar
+  });
+  // printing id on console
+  // eslint-disable-next-line no-console
+  console.log(res.data);
```
Comment (kole-swapnil) [inline javascripts/applicantDummyData.js:33]: pls remove all eslint 
use debug.log

File: javascripts/applicantDummyData.js [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+  const res = await axios.get('http://localhost:5000/api/v1/user/logout', {
+    jar: cookieJar, // tough.CookieJar or boolean
+    withCredentials: true, // If true, send cookie stored in jar
+  });
+  // eslint-disable-next-line no-console
+  console.log(res.data);
+}
+
+/* eslint-disable no-await-in-loop */
+async function sequentialCall() {
```
Comment (kole-swapnil) [inline javascripts/applicantDummyData.js:76]: same as before

**Review by kole-swapnil — CHANGES_REQUESTED** [review]

File: javascripts/applicantDummyData.js (hunk 6 above)
Comment (kole-swapnil) [reply javascripts/applicantDummyData.js:33]: *eslint-disable

File: javascripts/applicantDummyData.js (hunk 7 above)
Comment (ayush-raj13) [reply javascripts/applicantDummyData.js:74]: done

File: javascripts/applicantDummyData.js (hunk 6 above)
Comment (ayush-raj13) [reply javascripts/applicantDummyData.js]: done

**Review by kole-swapnil — APPROVED** [review]

**Review by Srutip04 — APPROVED** [review]

**Pull request merged**